apps/wallet/src/ui/app/components/receipt-card/index.tsx

```diff
@@ -144,17 +129,18 @@ function ReceiptCard({ txn, activeAddress }: ReceiptCardProps) {
                             />
                         )} */}
 
-                        {transferAmount && transferAmount.length > 0
-                            ? transferAmount.map(
-                                  ({ amount, coinType, receiverAddress }) => {
+                        {transferAmount &&
+                        transferAmount.balanceChanges?.length > 0
+                            ? transferAmount.balanceChanges.map(
+                                  ({ amount, coinType, address }) => {
                                       return (
                                           <div
-                                              key={coinType + receiverAddress}
+                                              key={coinType}
```

Comment:
There are instances where we can have the same coin type but multiple addresses.

Reply:
added `address` back to the key